app.json

```diff
@@ -55,6 +55,9 @@
     "MITXPRO_ADMIN_EMAIL": {
       "description": "E-mail to send 500 reports to."
     },
+    "MITXPRO_BASE_URL": {
```

Comment:
Take or leave, since this is an issue on our other codebases: this is a very general name for something that's only supposed to be used in mail templates. Wouldn't `MAILGUN_SITE_URL` et. al. make more sense?

Reply:
I'll update the description here, I copied this definition from discussions, and even there it's inaccurate as it's used in social auth and the frontend ([example in my next PR](https://github.com/mitodl/mitxpro/pull/54/files#diff-fcb8bea0ec866f7d62bb722d6632ef04R169))